prow/plugins/label/label.go

```diff
@@ -34,6 +34,7 @@ const pluginName = "label"
 
 var (
 	defaultLabels          = []string{"kind", "priority", "area"}
+	commentRegex           = regexp.MustCompile(`(?s)<!--(.*)-->`)
```

Comment:
probably use non-greedy matching, otherwise two consecutive comments could drop content between them?

```suggestion
	commentRegex           = regexp.MustCompile(`(?s)<!--(.*?)-->`)
```

Reply:
Sounds good, fixed :+1: